core/testdata/TestVirtualMachineBuild_JSONMarshaling/full.golden

```diff
@@ -1 +1 @@
-{"id":"id1","spec_xml":"<xml/>","state":"draft","virtual_machine":{"id":"id2"},"created_at":1600192008}
```

Comment:
Is this change in encoding anything to be concerned about?

Reply:
Interesting, I'm gonna have to do some testing around this.

I was under the impression that `json.Marshal()` did the same escape setup as the codec package does, but maybe I was imaging things... lol

Reply:
You may need to create a JSON encoder and use SetEscapeHTML(false). I've had to do this before.